score/scripts/generate_mbtiles.py

```diff
@@ -1,7 +1,31 @@
 import os
+from pathlib import Path
+import shutil
 
-# remove existing file
-os.remove("./data/tiles/block2010.mbtiles")
+data_path = Path.cwd().joinpath("data")
+
+# remove existing mbtiles file
+mb_tiles_path = data_path.joinpath("tiles", "block2010.mbtiles")
+if os.path.exists(mb_tiles_path):
+    os.remove(mb_tiles_path)
+
+# remove existing mvt directory
+mvt_tiles_path = data_path.joinpath("tiles", "mvt")
+if os.path.exists(mvt_tiles_path):
+    shutil.rmtree(mvt_tiles_path)
```

Comment:
pathlib gives a `.rmdir()` so you wouldn't need `shutil`

Reply:
Oh, I thought I tried it but it complained the directory wasn't empty? Maybe in Windows?

Reply:
Yup, this fails:

```
-> mvt_tiles_path.rmdir()
(Pdb) n
OSError: [WinError 145] The directory is not empty: 'C:\\opt\\justice40-tool\\score\\data\\tiles\\mvt'
> c:\opt\justice40-tool\score\scripts\generate_mbtiles.py(15)<module>()
-> mvt_tiles_path.rmdir()
```